x-pack/plugins/secops/public/components/timeline/timeline.tsx

```diff
@@ -111,19 +108,22 @@ export const Timeline = pure<Props>(
 
               <div data-test-subj="timeline">
                 {combinedQueries != null ? (
-                  <EventsQuery
+                  <TimelineQuery
                     sourceId="default"
-                    startDate={combinedQueries.queryProps.startDate}
-                    endDate={combinedQueries.queryProps.endDate}
-                    filterQuery={combinedQueries.queryProps.filterQuery}
+                    limit={itemsPerPage}
+                    filterQuery={combinedQueries.filterQuery}
+                    sortField={{
+                      sortFieldId: sort.columnId,
+                      direction: sort.sortDirection as Direction,
```

Comment:
I have seen this same `as Direction` cast used in a few other places and playing with it I think that they all should be typed as:

```ts
direction: Direction.descending
```

and that will remove another `as` cast and make it to where if it's changed to a different string you won't have to hunt and change all the occurrences.